src/features/auth/pages/signup/SetEmail/SetEmail.native.test.tsx

```diff
@@ -214,4 +214,18 @@ describe('<SetEmail />', () => {
     const marketingEmailSubscriptionCheckbox = screen.getByRole('checkbox')
     expect(marketingEmailSubscriptionCheckbox.props.accessibilityState.checked).toBe(true)
   })
+
+  it('should set a default marketing email subscription choice to false', () => {
+    const propsWithoutMarketingEmailSubscription = {
+      ...props,
+      previousSignupData: {
+        ...props.previousSignupData,
+        marketingEmailSubscription: undefined,
+      },
+    }
+    render(<SetEmail {...propsWithoutMarketingEmailSubscription} />)
+
+    const marketingEmailSubscriptionCheckbox = screen.getByRole('checkbox')
+    expect(marketingEmailSubscriptionCheckbox.props.accessibilityState.checked).toBe(false)
+  })
```

Comment:
Avec la MAJ de RNTL c'est mieux d'utiliser `expect(marketingEmailSubscriptionCheckbox).toBeChecked()` est-ce que ça fonctionne ici ? 🙂

Reply:
Non ça ne fonctionne pas :(

Reply:
je voulais dire `expect(marketingEmailSubscriptionCheckbox).not.toBeChecked()` mais j'imagine que c'est pareil... Je regarderai peut être ça en journée de pratique parce que RNTL déconseille vraiment de passer par les props mais on peut laisser comme ça ici pour creuser plus tard !

Reply:
C'est le toBeChecked() qui ne fonctionne pas :(